src/main/webapp/i18n/pt-pt/feedback.json

```diff
@@ -26,7 +26,8 @@
       "updated": "Sugestão de melhoria atualizada com sucesso",
       "deleted": "Sugestão de melhoria eliminada com sucesso",
       "delete": {
-        "question": "Tem a certeza que pretende eliminar esta sugestão de melhoria?"
+        "question": "Tem a certeza que pretende eliminar esta sugestão de melhoria?",
+        "body": "Ao confirmar esta ação, toda a informação relativa a este sugestão de melhoria será perdida."
```

Comment:
Please correct translations according to mock up
![image](https://user-images.githubusercontent.com/38863507/106193063-eafe5780-61a4-11eb-8ae8-64157c20cd70.png)

Reply:
@andrenns The were defined by @sfdsilva, So I presume they are correct and the mockeups are not updated.

See here: https://github.com/MSE-2019-2021/support_care/commit/350fc89410a1a454d535603176e8b58b1ef1bd6b